Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -936,6 +936,70 @@ def liftEquivOfComm :
       rfl }
 #align triv_sq_zero_ext.lift TrivSqZeroExt.liftEquiv
 
+section map
+
+variable {N P : Type*} [AddCommMonoid N] [Module R' N] [Module R'ᵐᵒᵖ N] [IsCentralScalar R' N]
+  [AddCommMonoid P] [Module R' P] [Module R'ᵐᵒᵖ P] [IsCentralScalar R' P]
+
+/-- Functoriality of `TrivSqZeroExt` when the ring is commutative: a linear map
+`f : M →ₗ[R'] N` induces a morphism of `R'`-algebras from `TrivSqZeroExt R' M` to
+`TrivSqZeroExt R' N`.
+
+Note that we cannot neatly state the non-commutative case, as we do not have morphisms of bimodules.
+-/
+def map (f : M →ₗ[R'] N) : TrivSqZeroExt R' M →ₐ[R'] TrivSqZeroExt R' N :=
+  liftEquivOfComm ⟨inrHom R' N ∘ₗ f, fun _ _ => inr_mul_inr _ _ _⟩
```

Comment:
Please mention this in the commit message

Reply:
I updated the commit message.